pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -105,24 +107,32 @@ func NewDiscoveryServer(env *model.Environment, generator core.ConfigGenerator)
 
 	out.DebugConfigs = os.Getenv("PILOT_DEBUG_ADSZ_CONFIG") == "1"
 
-	pushThrottle := intEnv("PILOT_PUSH_THROTTLE", 10)
-	pushBurst := intEnv("PILOT_PUSH_BURST", 100)
+	pushThrottle := intEnv("PILOT_PUSH_THROTTLE", 25)
+	pushBurst := intEnv("PILOT_PUSH_BURST", 200)
 
 	adsLog.Infof("Starting ADS server with throttle=%d burst=%d", pushThrottle, pushBurst)
-	rate := time.Second / time.Duration(pushThrottle)
-	burstLimit := pushBurst
-	tick := time.NewTicker(rate)
-	out.throttle = make(chan time.Time, burstLimit)
+
+	// init throttle affects new connections from sidecars.
+	out.initThrottle = initThrottle("initConnection", pushBurst, pushThrottle*2)
```

Comment:
looking at the logic, this should be a backlog size of 25.. We allow new sidecars as we clear out old ones.